src/services/balancer/pools/exits/handlers/generalised-exit.handler.ts

```diff
@@ -16,21 +16,13 @@ import { flatTokenTree } from '@/composables/usePool';
 import { getAddress } from '@ethersproject/address';
 import { TransactionBuilder } from '@/services/web3/transactions/transaction.builder';
 
-interface GeneralisedExitResponse {
-  to: string;
-  callData: string;
-  tokensOut: string[];
-  expectedAmountsOut: string[];
-  minAmountsOut: string[];
-  priceImpact: string;
-}
+type ExitResponse = Awaited<ReturnType<typeof balancer.pools.generalisedExit>>;
```

Comment:
This is cool! Didn't know you can build types like this. I just had a read through all the Typescript utility types.